CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### API-Changes
 
 - Add `dc_jsonrpc_blocking_call()`.
+- Add rust-only api `get_status_update`.
```

Comment:
That's the changelog of the previous release, which shouldn't be edited.

Instead:
- Call this PR `api: Add rust-only api `get_status_update`.
- When merging the PR, squash merge it into a single commit with this commit message.

This recently changed. See CONTRIBUTING.md for details.